src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,211 @@
+    </li>
+    <li>
+      <b>For large changes that include multiple components, please submit changes in several small pull requests instead of one large pull request.</b> (Large being more than 50 files). We ask that you do this for the following reasons:
+    </li>
+    <ul>
+      <li>
+        It allows the SKY UX team to review your code in manageable pieces. It's much easier to review 3 pull requests with 50 files than 1 pull request with 150 files.
+      </li>
+      <li>
+        It allows you to get feedback sooner. Catching pervasive coding style changes in a small code review allows you update the rest of your code before you submit it.
+      </li>
+      <li>
+        It demonstrates that your code is modular and does not contain excessive interdependencies. For instance, if Component B is a child component of Component A, you should be able to author, test, document, review, and release Component B in the absence of Component A. If your changes must include both Component A and Component B to be tested and reviewed, then it raises a red flag that the components may be too tightly coupled and should be refactored.
+      </li>
+    </ul>
+    <li>
+      <b>If you make an API change that affects inputs and outputs for a component, we recommend that you provide your potential (not working) documentation and demo to the SKY UX team for review.</b> This lets us head off API issues before implementation. Contributors should keep in mind that the API can change in the course of development.
+    </li>
+  </ul>
+
+  <stache-page-anchor>
+    Localization
+  </stache-page-anchor>
+
+  <p>
+    To declare localization strings, specify a string name and provide the string and a description in the <stache-code>resources_en_US.json</stache-code> file in <stache-code>src/assets/locales</stache-code>. You can reference the localization strings with the <stache-code>skyLibResources</stache-code> pipe or the <stache-code>SkyLibResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+    </p>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the repo</a> into your own GitHub repo.
```

Comment:
It seems like we should have a second sentence in this step along the lines of To view a list of SKY UX repositories that target different areas of development, see the packages matrix." and link to https://developer.blackbaud.com/skyux/contribute/packages. What do you think?